homepage.css

```diff
@@ -0,0 +1,399 @@
+	scroll-behavior: auto;
+}
+
+/*Carousel items*/
+.carousel .card {
+	position: relative;
+	width: 200px;
+	/* Fixed width */
+	height: 200px;
+	/* Fixed height */
+	margin: 10px;
+	border: 1px solid #ccc;
+	border-radius: 4px;
+	box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+	overflow: hidden;
+}
+
+.carousel .card .img {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+	border: 4px solid #fff;
+}
+
+.carousel .card h2 {
+	font-weight: 500;
+	font-size: 1.56rem;
+	margin: 30px 0 5px;
+}
+
+.carousel .card span {
+	color: #6A6D78;
+	font-size: 1.31rem;
+}
+
+.carousel .card .boxcontent {
+	position: absolute;
+	bottom: 10%;
+	left: 0;
+	right: 0;
+	padding: 15px;
+	background-color: #edffef;
+	color: #182f32;
+	text-align: center;
+	opacity: 0;
+	transition: opacity 0.3s;
+}
+
+.carousel .card:hover .boxcontent {
+	opacity: 1;
+}
+
+.carousel .card h3 {
+	font-size: 20px;
+	margin-top: 0;
+	margin-bottom: 10px;
+	color: #213032;
+}
+
+.card p {
+	font-size: 14px;
+	margin-bottom: 10px;
+}
+
+.carousel .card button {
+	position: relative;
+	background-color: #213032;
+	color: #fff;
+	text-decoration: none;
+	border: 2px solid black;
+	font-weight: bold;
+	transition: background-color 0.4s, color 0.4s, border 0.4s;
+	padding: 13px 30px;
+}
+
+.carousel .card button:hover {
+	background-color: #97a2a3;
+	color: #213032;
+	border: 2px solid black;
+	cursor: pointer;
+}
+
+.carousel .view {
+	display: inline-block;
+	padding: 10px 20px;
+	background-color: black;
+	color: #fff;
+	text-decoration: none;
+	border-radius: 5px;
+	border: none;
+	cursor: pointer;
+	transition: background-color 0.3s;
+}
+
+.carousel .view:hover {
+	background-color: grey;
+}
+
+/*All Recipes Section General CSS*/
+.allrecipes {
+	width: 100%;
+	padding: 30px 0px 30px;
+	background-color: #314e52;
+	margin: 0px;
+}
+
+/*All recipes section CSS for the boxes*/
+.box {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+	justify-content: center;
+}
+
+.card {
+	position: relative;
+	width: 200px;
+	/* Fixed width */
+	height: 200px;
+	/* Fixed height */
+	margin: 10px;
+	border: 2px solid black;
+	border-radius: 50%;
+	/* Make border-radius 50% to create a circular shape */
+	box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+	overflow: hidden;
+}
+
+.box .card img {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+}
+
+.box .card .boxcontent {
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	padding: 40px;
+	background-color: #edffef;
+	color: #182f32;
+	text-align: center;
+	opacity: 0;
+	transition: opacity 0.3s;
+}
+
+.box .card:hover {
+	transform: scale(1.2)
+}
+
+.box .card:hover .boxcontent {
+	opacity: 1;
+}
+
+.card h3 {
+	font-size: 20px;
+	margin-top: 0;
+	margin-bottom: 10px;
+	color: #213032;
+}
+
+.box .card p {
+	font-size: 14px;
+	margin-bottom: 10px;
+}
+
+.box .card button {
+	position: relative;
+	background-color: #213032;
+	color: #fff;
+	text-decoration: none;
+	border: 2px solid black;
+	font-weight: bold;
+	transform: .4s;
```

Comment:
Transform value error. Need to be a number i.e. 0.4

Reply:
Amended transform to 1s